src/components/pages/contact/leaflet/leaflet.tsx

```diff
@@ -57,13 +62,23 @@ const MapWrapper = styled.div`
   position: relative;
 `;
 
+const EnableGestureHandling = () => {
+  /**
+   * leaflet-gesture-handling does not add gestureHandling to the types
+   * but map.gestureHandling exists and works
+   */
+  const map = useMap() as any;
+  map.gestureHandling.enable();
+  L.Map.addInitHook('addHandler', 'gestureHandling', GestureHandling);
+  return null;
```

Comment:
You can put this method directly into the `Leaflet` component. It's usually nice to put features into own components, but this time it looks like too much.

Reply:
I tried to do so, but useMap() can only be used in child components of the MapContainer, that is why it was not possible to do so